tests/v2/validation/projects/rbac_terminating_project_test.go

```diff
@@ -0,0 +1,150 @@
+	err = checkPodLogsForErrors(pr.client, projectsApi.LocalCluster, leaderPodName, projectsApi.RancherNamespace, errorRegex, logCaptureStartTime)
+	require.NoError(pr.T(), err)
+
+	logCaptureStartTime = time.Now()
+	log.Info("Remove the finalizer that was previously added to the project.")
+	finalizer = nil
+	_, err = updateProjectNamespaceFinalizer(pr.client, updatedProject, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
+
+	log.Info("Verify that there are no errors in the Rancher logs related to role binding.")
+	err = checkPodLogsForErrors(pr.client, projectsApi.LocalCluster, leaderPodName, projectsApi.RancherNamespace, errorRegex, logCaptureStartTime)
+	require.NoError(pr.T(), err)
+}
+
+func (pr *RbacTerminatingProjectTestSuite) TestUserAdditionToProjectWithTerminatingProjectNamespace() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a standard user.")
+	createdUser, err := users.CreateUserWithRole(pr.client, users.UserConfig(), projectsApi.StandardUser)
+	require.NoError(pr.T(), err)
+	pr.T().Logf("Created user: %v", createdUser.Username)
+
+	log.Info("Create a project in the downstream cluster.")
+	createdProject, err := createProject(pr.client, pr.cluster.ID)
+	require.NoError(pr.T(), err)
+	err = waitForFinalizerToUpdate(pr.client, createdProject.Name, createdProject.Namespace, 2)
+	require.NoError(pr.T(), err)
+
+	log.Info("Simulate a project stuck in terminating state by adding a finalizer to the project.")
+	finalizer := append([]string{dummyFinalizer}, createdProject.Finalizers...)
+	updatedProject, err := updateProjectNamespaceFinalizer(pr.client, createdProject, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
+	err = waitForFinalizerToUpdate(pr.client, createdProject.Name, createdProject.Namespace, 3)
+	require.NoError(pr.T(), err)
+
+	log.Info("Delete the Project.")
+	err = projectsApi.DeleteProject(pr.client, createdProject.Namespace, createdProject.Name)
+	require.Error(pr.T(), err)
+	err = waitForFinalizerToUpdate(pr.client, createdProject.Name, createdProject.Namespace, 1)
+	require.NoError(pr.T(), err)
+
+	log.Info("Add the standard user to the project as project owner.")
+	_, err = createProjectRoleTemplateBinding(pr.client, createdUser, createdProject, roleProjectOwner)
+	require.Error(pr.T(), err)
+	prtbNamePlaceholder := `[^"]+`
+	regexPattern := fmt.Sprintf(`projectroletemplatebindings\.management\.cattle\.io "%s" is forbidden: unable to create new content in namespace %s because it is being terminated`, prtbNamePlaceholder, regexp.QuoteMeta(createdProject.Name))
+	expectedErrorMessage := regexp.MustCompile(regexPattern)
+	require.Regexp(pr.T(), expectedErrorMessage, err.Error())
+
+	log.Info("Remove the finalizer that was previously added to the project.")
+	finalizer = nil
+	_, err = updateProjectNamespaceFinalizer(pr.client, updatedProject, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
```

Comment:
Since we are deleting the finalizer, can we rename this to Failed to deleted fializer?

Reply:
Renamed it to "Failed to remove the finalizer"